Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/AnimGraph/AnimGraphPlugin.h

```diff
@@ -63,6 +65,8 @@ namespace EMStudio
     class TimeViewPlugin;
     class NavigationHistory;
 
+    static constexpr GraphCanvas::EditorId AnimGraphEditorId = AZ_CRC("AnimGraphEditor",  0xf44fce98);
```

Comment:
Nit: Let the compiler compute the hash for us

```suggestion
    static constexpr GraphCanvas::EditorId AnimGraphEditorId = AZ_CRC_CE("AnimGraphEditor");
```

Reply:
Actually the hash was a mismatch which failed the validation on AR, so even more of a reason to use `AZ_CRC_CE` instead :)

```
VALIDATOR_FAILED: CrcValidator /data/workspace/o3de/Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/AnimGraph/AnimGraphPlugin.h::CrcValidator FAILED - Source file contains a CRC mismatch!
[2022-08-29T18:10:46.857Z]          AZ_CRC("AnimGraphEditor", 0xf44fce98), expected value 0xfdba7ae7
```